boot/modeenv.go

```diff
@@ -62,6 +66,10 @@ type Modeenv struct {
 	// asset names to a list of hashes of the asset contents. Used similarly
 	// to CurrentTrustedBootAssets.
 	CurrentTrustedRecoveryBootAssets bootAssetsMap `key:"current_trusted_recovery_boot_assets"`
+	// KernelCommandLines is stack of the expected kernel command lines when
+	// booting into run mode.
+	KernelCommandLines bootCommandLines `key:"kernel_command_lines"`
```

Comment:
for consistency with the rest this should be called current_kernel_command_lines I think